pkgs/tools/backup/git-sync/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, coreutils, gnugrep, gnused, makeWrapper, git
+}:
+
+stdenv.mkDerivation rec {
+  name = "git-sync";
```

Comment:
The version should be part of the name like `"name-${version}"`. In the case of a project like this with no releases, we typically use the date the commit was made Oct 24, 2015, so we'd use `20151024` as the version. This would let us have increasing version numbers when we make updates to the package. (Assuming updates are continuing for this package!)

Reply:
The version should probably be `unstable-2015-10-24`, see https://nixos.org/nixpkgs/manual/#sec-package-naming.